buddybuild_postbuild.sh

```diff
@@ -0,0 +1,10 @@
+set -o pipefail
+
+echo Destination: iPhone X
+xcodebuild -scheme PusherPlatform -destination "name=iPhone X" test | xcpretty
+echo Destination: Apple Watch Series 3 - 42mm
+xcodebuild -scheme PusherPlatform -destination "name=Apple Watch Series 3 - 42mm" test | xcpretty
```

Comment:
AFAIK you can't `test` with Apple Watch as the destination. I don't think it has access to XCTest. The best you can do is just test that it builds successfully.

Reply:
No, it's still the same 👍